.github/workflows/test.yaml

```diff
@@ -20,11 +20,13 @@ jobs:
       # do not abort the whole test job if one combination in the matrix fails
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
-        os: [ubuntu-20.04, macos-latest, windows-latest]
-        exclude:
-          - os: macos-latest
-            python-version: "3.6"
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        os: [ubuntu-20.04]
+        include:
+          - python-version: "3.7"
+            os: macos-latest
+          - python-version: "3.7"
+            os: windows-latest
```

Comment:
Now we're not testing all Python versions on Windows and Mac, but I don't think this is an issue, right?

Reply:
Yes this is intentional. I just wanted to use fewer resources.